### PR TITLE
PMM 12408 nodes link inventory

### PR DESCRIPTION
**Pull request by rasika-chivate**
For Details refer to:

https://jira.percona.com/browse/PMM-11963

### CODE REVIEW

**Comment by rnovikovP** [comment]
@matejkubinec , is this feature also cover both cases: Services->nodes and Nodes->services linking? 
I tink we need more explanation and images updates here

**Comment by matejkubinec** [comment]
@rnovikovP yes, it's both Service -> Node and Node -> Services.

File: docs/details/dashboards/dashboard-inventory.md [hunk 1]
```diff
@@ -8,11 +8,23 @@ To check your inventory list, go to <i class="uil uil-cog"></i> **Configuration*
 
 Inventory objects form a hierarchy with Node at the top, then Service and Agents assigned to a Node. This information is detailed in the two tabs available on this page.
 
-### **Services** tab
+### Services tab
+
+The **Services** tab displays the individual services, the nodes on which they run, and the Agents that help collect the service metrics along with the following information:
+
+**Status** - The **Status** column shows the status of your databases based on metrics coming directly from the database.
```
Comment (matejkubinec) [inline docs/details/dashboards/dashboard-inventory.md:15]: I would remove `The Status column` as it's probably redundant.

File: docs/details/dashboards/dashboard-inventory.md [hunk 2]
```diff
@@ -8,11 +8,23 @@ To check your inventory list, go to <i class="uil uil-cog"></i> **Configuration*
...
+
+**Service name** -  The name or identifier associated with the service being monitored.
+
+**Node Name** - Name or identifier associated with a specific node. 
+
+**Monitoring status** - Is the service currently being monitored, or has it failed.
```
Comment (matejkubinec) [inline docs/details/dashboards/dashboard-inventory.md:21]: I think the previous description was better as it relates to agents statuses.

File: docs/details/dashboards/dashboard-inventory.md [hunk 3]
```diff
@@ -31,6 +43,19 @@ Each binary (exporter, agent) running on a client will get an `agent_type` value
 To view the agents running on a service and their health status, click **OK** or **Failed** under the **Monitoring** column. Furthermore, you can also check the properties of a particular agent by clicking the <image src="../../_images/arrow-downward.ico" width="15px" aria-label="downward arrow"/> icon under the **Options** column.
 ![!image](../../_images/PMM_Inventory_Service_Agent_Properties.png)
 
+#### Node-service relationship
+
+Starting with PMM 2.40.0, you click on the link in the **Node Name** column to view the node on which a specific service is running and analyze how node-level resource utilization impacts the performance of those services.
```
Comment (matejkubinec) [inline docs/details/dashboards/dashboard-inventory.md:48]: you *can* click

or

clicking on the link in the Node Name column will show ...

File: docs/details/dashboards/dashboard-inventory.md [hunk 4]
```diff
@@ -42,6 +67,8 @@ By expanding the entry from the options column, you can check the node labels an
 
 Starting with PMM 2.38.0, you can see the number of agents running on any particular node. When you click on any node, the UI navigates to the view of agents, which is filtered to display only agents related to that specific node. 
 
+Also, starting with PMM 2.40.0, when you click on the link in the **Services** column, you can see the services that are running on that specific node.
```
Comment (matejkubinec) [inline docs/details/dashboards/dashboard-inventory.md:70]: You will only see that one particular service (which the name belongs to) not all of the services running on that node.

File: docs/details/dashboards/dashboard-inventory.md (hunk 1 above)
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-inventory.md:15]: Done.

File: docs/details/dashboards/dashboard-inventory.md (hunk 2 above)
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-inventory.md:21]: Will fix it.

File: docs/details/dashboards/dashboard-inventory.md (hunk 3 above)
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-inventory.md:48]: Fixed.

File: docs/details/dashboards/dashboard-inventory.md (hunk 4 above)
Comment (rasika-chivate) [reply docs/details/dashboards/dashboard-inventory.md:70]: Done.

**Review by rnovikovP — APPROVED** [review]

**Review by matejkubinec — APPROVED** [review]

**Pull request merged**